src/api/modelmapper.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+from pathlib import Path
+
+USE_MODEL_MAPPING = os.getenv("USE_MODEL_MAPPING", "true").lower() == "true"
```

Comment:
Could we make this the default and maybe you can disable it with an env var? It will mean less boilerplate for the use case that we will promote in the documentation

Reply:
Naming: this ALL_CAPS global var is not very python

Reply:
```suggestion
USE_MODEL_MAPPING = os.getenv("USE_MODEL_MAPPING", "true").lower() in ("true", "1")
```

Reply:
Move this to `setting.py`

Reply:
cf. the other settings
```suggestion
USE_MODEL_MAPPING = os.getenv("USE_MODEL_MAPPING", "true").lower() != "false"
```